libs/ui/src/ListBox/ListBox.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { render, screen } from '@testing-library/react'
+
+import { ListBox } from './ListBox'
+
+describe('ListBox', () => {
+  it('should pass classnames properly', () => {
+    const { rerender } = render(<ListBox />)
+
+    const ul = screen.getByRole('list')
+
+    expect(ul).toHaveClass('_list_c59894')
+
+    rerender(<ListBox className="test-123" />)
```

Comment:
This rerender isn't really necessary I think, you just want to test if the classname is passed, no?

Reply:
Changed it a bit, you also want to check it doesnt add a class "undefined"

Reply:
Changed the test a bit.